### DISCUSSION TITLE
What is Riju's Release Schedule?

### DISCUSSION THREAD

**Discussion by JasonSteving99** [raxod502/riju#146] [Q&A]
Hi @raxod502, thank you again for merging Claro into Riju. I'm curious to ask how often Riju redeploys, or at least reinstalls languages? In my case this would be to pick up the newest releases in my github repo. Obviously I'm more than happy with anything best effort, I'm not expecting a strict schedule, but it'd be useful to know a ballpark of if/when new Claro features would reach riju.codes/claro to play with after I implement them. And is there anywhere I can go to see explicitly that a rebuild/redeploy has happened?

**Comment by raxod502** [comment]
Unfortunately, there is not a good story for this at present. In particular, there is no automation, so updating a language is a fully manual process. Part of my plans for a rearchitecting of Riju include support for community-submitted language updates which would only have to be approved through a web interface rather than compiled and uploaded from my laptop. In addition, unfortunately there is no information exposed at present about what revision is live, or when it was produced. This issue would also be resolved by the aforementioned rearchitecture.

But what is really needed to provide these features is a database-backed system for managing language artifacts, and the current system simply doesn't have that.
